assets/css/global-custom.css

```diff
@@ -477,6 +477,10 @@ body {
     padding-right: 30px;
 }
 
+.faq-table{
```

Comment:
We don't want this right? Remove this @janithlahirukariyawasam

Reply:
Ah yes!

Reply:
Thank you again Master Mind!

Reply:
Any time @janithlahirukariyawasam